ros_buildfarm/config/doc_build_file.py

```diff
@@ -17,7 +17,9 @@
 DOC_TYPE_ROSDOC = 'rosdoc_lite'
 DOC_TYPE_MANIFEST = 'released_manifest'
 DOC_TYPE_MAKE = 'make_target'
-DOC_TYPES = [DOC_TYPE_ROSDOC, DOC_TYPE_MANIFEST, DOC_TYPE_MAKE]
+DOC_TYPE_EXTERNAL = 'external_site'
+DOC_TYPES = [DOC_TYPE_ROSDOC, DOC_TYPE_MANIFEST,
```

Comment:
To avoid aligning please wrap after the open bracket.

Reply:
Sure. Done.